mindsdb/integrations/handlers/google_docs_handler/README.md

```diff
@@ -0,0 +1,58 @@
+# Google Docs API Integration
+This handler integrates with the Google Docs API to make the docs content available.
+
+# Connect to the Google Docs API 
+First we need to create a database to connect to the Google Docs API.
+
+However, you will need to have a Google account and have enabled the Google Docs API and scope of the API can be restricted to read only. More information on the scope of the API can be found [here](https://developers.google.com/identity/protocols/oauth2/scopes#docs)
+Also, you will need to have a doc id from your google docs and the credentials for that google doc in a json file. credentials.json file can be created using the link [here](https://developers.google.com/workspace/guides/create-credentials#oauth-client-id)
```

Comment:
Hey @bala-ceg,
I think it might be useful for us to mention how to retrieve the document ID here? I believe it is from the URL?

Reply:
Yes mentioned this one in the latest change.